bpmsuite/bpmsuite70-executionserver-postgresql-persistent.json

```diff
@@ -0,0 +1,680 @@
+            "required": false
+        },
+        {
+            "displayName": "KIE Server Bypass Auth User",
+            "description": "KIE execution server bypass auth user (Sets the org.kie.server.bypass.auth.user system property)",
+            "name": "KIE_SERVER_BYPASS_AUTH_USER",
+            "value": "false",
+            "required": false
+        },
+        {
+            "displayName": "KIE Server Persistence Dialect",
+            "description": "KIE execution server persistence dialect (Sets the org.kie.server.persistence.dialect system property)",
+            "name": "KIE_SERVER_PERSISTENCE_DIALECT",
+            "value": "org.hibernate.dialect.PostgreSQLDialect",
+            "required": false
+        },
+        {
+            "displayName": "KIE Server Persistence DS",
+            "description": "KIE execution server persistence datasource (Sets the org.kie.server.persistence.ds system property)",
+            "name": "KIE_SERVER_PERSISTENCE_DS",
+            "value": "java:/jboss/datasources/ExampleDS",
+            "required": false
+        },
+        {
+            "displayName": "KIE Server Persistence TM",
+            "description": "KIE execution server persistence transaction manager (Sets the org.kie.server.persistence.tm system property)",
+            "name": "KIE_SERVER_PERSISTENCE_TM",
+            "value": "org.hibernate.engine.transaction.jta.platform.internal.JBossAppServerJtaPlatform",
+            "required": false
+        },
+        {
+            "displayName": "KIE MBeans",
+            "description": "KIE execution server mbeans enabled/disabled (Sets the kie.mbeans and kie.scanner.mbeans system properties)",
+            "name": "KIE_MBEANS",
+            "value": "enabled",
+            "required": false
+        },
+        {
+            "displayName": "Drools Server Filter Classes",
+            "description": "KIE execution server class filtering (Sets the org.drools.server.filter.classes system property)",
+            "name": "DROOLS_SERVER_FILTER_CLASSES",
+            "value": "true",
+            "required": false
+        },
+        {
+            "displayName": "Application Name",
+            "description": "The name for the application.",
+            "name": "APPLICATION_NAME",
+            "value": "myapp",
+            "required": true
+        },
+        {
+            "displayName": "Execution Server Custom http Route Hostname",
+            "description": "Custom hostname for http service route. Leave blank for default hostname, e.g.: <application-name>-execserv-<project>.<default-domain-suffix>",
+            "name": "EXECUTION_SERVER_HOSTNAME_HTTP",
+            "value": "",
+            "required": false
+        },
+        {
+            "displayName": "Database JNDI Name",
+            "description": "Database JNDI name used by application to resolve the datasource, e.g. java:/jboss/datasources/ExampleDS",
+            "name": "DB_JNDI",
```

Comment:
What is the difference between KIE_SERVER_PERSISTENCE_DS and DB_JNDI? AFAIK it should point to the same value so it would be good to be the same property. What do you think?

Reply:
@rbarriuso @errantepiphany What is the point of defining KIE_SERVER_PERSISTENCE_DS when DB_JNDI seems to cover the functionality?

Reply:
@rbarriuso Quartz configuration has not yet been added to the bpmsuite7 executionserver image. @sutaakar I see your point, and we can probably change this to default to DB_JNDI, or just use it instead. Can you please open up a separate jira for it, and assign it to me?

Reply:
@errantepiphany Sure, https://issues.jboss.org/browse/GUVNOR-3437

Reply:
@sutaakar Thanks!